### PR TITLE
Added test/runtests.jl and added to .gitignore

### PR DESCRIPTION
**Pull request by charleskawczynski**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -0,0 +1,54 @@
+using Test
+
+function is_test_folder(f, path_separator)
+  a = split(f, path_separator)
+  dir = join(a[1:end-1], path_separator)
```
Comment (lcw) [inline test/runtests.jl]: You may want to look at `joinpath` it takes care of the path separator for you.

File: test/runtests.jl [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  d = split(dir, path_separator)[end]
```
Comment (lcw) [inline test/runtests.jl]: There will be a `splitpath` in future versions of julia https://github.com/JuliaLang/julia/issues/24477

File: test/runtests.jl (hunk 1 above)
Comment (charleskawczynski) [reply test/runtests.jl]: Ah, ok, I'll update.

File: test/runtests.jl (hunk 2 above)
Comment (charleskawczynski) [reply test/runtests.jl]: Thanks, I wasn't crazy about this function-- I'll update this in my next PR.

Comment (lcw) [reply test/runtests.jl]: no problem; also see the `isdir` function.

Comment (lcw) [reply test/runtests.jl]: also `basename`

**Pull request merged**